example/setMultiZoneEffect.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/**
+ * Searches for new lights, if one is found it sends a setMultiZoneEffect packet
+ */
+
+const LifxClient = require('../lib/lifx').Client;
+const packet = require('../lib/lifx').packet;
+const constants = require('../lib/lifx').constants;
+const client = new LifxClient();
+
+// Create our packet with for move effect - runs infinite
+const packetObj = packet.create('setMultiZoneEffect', {
+  effect_type: lifx_constants.MULTIZONE_EFFECTS.indexOf('MOVE'), // MOVE or OFF
+  speed: 1000, // 1s - speed of one full animation cycle
+  parameter2: lifx_constants.MULTIZONE_EFFECTS_MOVE_DIRECTION.indexOf(args.direction) // TOWARDS or AWAY from controller
+}, client.source);
+
+// Function running when packet was received by light
+const callback = function() {
```

Comment:
Use an arrow function here already as we'll move on to those soonish anyway when my PR will get merged. Rename to `onPacketReceived`?

Reply:
Not sure what you mean. const onPacketReceived = () => {?

Reply:
Should be fixed. Please crosscheck.